ui/imports/shared/views/profile/ProfileShowcaseView.qml

```diff
@@ -0,0 +1,384 @@
+                            }
+                        }
+                    }
+
+                    StatusToolTip {
+                        visible: hhandler.hovered && (!!model.name || !!model.description)
+                        text: {
+                            let name = model.name
+                            let descr = model.description
+                            let sep = !!name && !!descr ? "<br>" : ""
+                            return `<b>${name}</b>${sep}${descr}`
+                        }
+                    }
+
+                    HoverHandler {
+                        id: hhandler
+                        cursorShape: hovered ? Qt.PointingHandCursor : undefined
+                    }
+
+                    TapHandler {
+                        onSingleTapped: {
+                            Global.openLink(model.permalink)
+                        }
+                    }
+                }
+            }
+        }
+
+        // assets/tokens
+        ColumnLayout {
+            StatusBaseText {
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+                Layout.alignment: Qt.AlignCenter
+                visible: assetsView.count == 0
+                horizontalAlignment: Text.AlignHCenter
+                verticalAlignment: Text.AlignVCenter
+                color: Theme.palette.directColor1
+                text: qsTr("%1 doesn't have any assets/tokens yet").arg(d.mainDisplayName)
+            }
+
+            StatusGridView {
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+                id: assetsView
+                rightMargin: Style.current.halfPadding
+                cellWidth: (width-rightMargin)/3
+                cellHeight: cellWidth/2.5
+                visible: count
+                model: SortFilterProxyModel {
+                    // TODO show assets for all accounts, not just the current one?
+                    sourceModel: root.isCurrentUser ? root.walletStore.currentAccount.assets : null // TODO show other users too
+                    filters: ValueFilter {
+                        roleName: "visibleForNetworkWithPositiveBalance"
+                        value: true
+                    }
+                    sorters: [
+                        StringSorter {
+                            roleName: "name"
+                        },
+                        StringSorter {
+                            roleName: "symbol"
+                        }
+                    ]
+                }
+                ScrollBar.vertical: StatusScrollBar { }
+                delegate: StatusListItem {
+                    readonly property double changePct24hour: model.changePct24hour ?? 0
+                    readonly property string textColor: Math.sign(changePct24hour) === 0
+                                                        ? Theme.palette.baseColor1 : Math.sign(changePct24hour) === -1
```

Comment:
why not just
```suggestion
                                                        ? Theme.palette.baseColor1 : changePct24hour)  < 0
```
?

Reply:
This is old wallet code, as I said, used to be a string; will fix